### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by CannibalChoco**
Doc improvements after feedback from Alan.

### CODE REVIEW

**Review by 256p — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,13 +26,14 @@ The data from the library is then accessible via your account on the Pubguard da
...
 
-These instructions will enable you to get the Pubguard library running on your Android app.  
+The following instructions will enable you to get the Pubguard library running on your Android app.  
 - iOS instructions can be found [here](https://github.com/bidstack-group/pubguard-sdk-ios)
 - Unity instructions can be found [here](https://github.com/bidstack-group/pubguard-unity-plugin)
 
 ## Prerequisites
 
-Before installing the Pubguard library you will need an application key that is available from your account, to receive an API key please sign up from http://dashboard.pubguard.com or email support@pubguard.com. The key is used in both the iOS and Android installations.
+Before installing the Pubguard library you will need an API key which is available through your Pubguard account. To retrieve an API key, please access http://dashboard.pubguard.com or email support@pubguard.com. The key is used in both the iOS and Android installations. Please note that separate keys are used for Android and iOS installations.
```
Comment (alanhoilun) [inline README.md:35]: iOS and Android installations, please note that separate keys are used for Android Google Play and iOS installations.

File: README.md [hunk 2]
```diff
@@ -188,7 +189,7 @@ For all release notes and previous versions please see our [changelog](ChangeLog
 
 ### Advertising SDK support
 
-These are SDKs designed specifically for serving advertising content into your app, if you would like information on a version or vendor that is not on this list please email support@pubguard.com
+These are SDKs specifically supported by Pubguard to analyse advertising content in your app, if you would like information on a version or vendor that is not on this list please email support@pubguard.com
```
Comment (alanhoilun) [inline README.md:192]: These are ad network SDKs specifically supported by Pubguard

File: README.md (hunk 1 above)
Comment (CannibalChoco) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (CannibalChoco) [reply README.md]: fixed

**Pull request merged**